### PR TITLE
LWMIQ pictures are worth a thousand words

### PR DESCRIPTION
**Pull request by syncrou**

### CODE REVIEW

**Comment by syncrou** [comment]
@JPrause - This weeks' picture of what is going on around here.

**Comment by JPrause** [comment]
If possible, @lgalis @jntullo @imtayadeway can you each have a look as well.

**Comment by JPrause** [comment]
LGTM! It's like a football playbook! Ha! Cool idea. Always good to change the presentation of information.

**Comment by jntullo** [comment]
Creative approach! 🎉 

Only suggestion would be to add headers in the Providers section to break up the images, because it's a little busy with one picture after the other. Otherwise LGTM 👍

**Comment by syncrou** [comment]
Thanks @jntullo! - I pushed those changes.

**Comment by lgalis** [comment]
@syncrou - looks great!!

**Comment by JPrause** [comment]
@cybette or @hayesr looks like we're good to go.
Please merge. This is dated to pop tomorrow,...so no rush.

File: site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: "Last Week in ManageIQ: A picture is worth a thousand words"
+author: syncrou
+date: 2017-07-04
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi All, [syncrou](https://github.com/syncrou) here to welcome you to another round of Last Week in ManageiQ.
```
Comment (imtayadeway) [inline site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md]: Think you want a s/ManageiQ/ManageIQ/ here

File: site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Since we're in the throws of summer, I am the last *weekly* writer of Last Week in ManageIQ until the fall.
+
+Over the next few months we will be writing *bi-weekly* entries to keep you in the loop regarding of all the happenings
```
Comment (imtayadeway) [inline site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md]: Possibly "regarding all the happenings"?

File: site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md (hunk 1 above)
Comment (JPrause) [inline site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md]: Nice catch Tim!

File: site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+3. [Service UI][manageiq-ui-service PRs merged]
+
+4. [Provider Amazon][manageiq-providers-amazon PRs merged]
+
+5. [Provider Ansible Tower][manageiq-providers-ansible_tower PRs merged]
+
+I hope everyone has a great week!  Come back in two weeks and we'll update you with all the rousing happenings here at Last Week in ManageIQ.
+
+
+[manageiq PRs merged]: https://github.com/ManageIQ/manageiq/pulls?page=1&q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222017-06-19+..+2017-06-25%22+sort%3Acreated-desc&utf8=%E2%9C%93
```
Comment (imtayadeway) [inline site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md]: Looks like these dates are for the week before?

**Comment by imtayadeway** [comment]
Couple of comments/questions above but overall love it! ❤️

**Comment by JPrause** [comment]
@syncrou  once you finish the suggested edits from Tim,...we'll see about that merge.

File: site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md (hunk 3 above)
Comment (cybette) [inline site/_posts/2017-06-30-pictures-are-worth-a-thousand-words.md]: I thought the PR counts looked familiar... :anguished:

**Comment by cybette** [comment]
Fun entry @syncrou! You can also consider uploading the images to /site/assets/images/blog/ if you like.

**Comment by syncrou** [comment]
@cybette, @imtayadeway - Dates and other suggested items are fixed.  I ended up having to redo a couple of the images after the dates were updated - Those are finished too.

The provider images are still from last week - as in all cases there were no new commits - so I decided to leave them.

Let me know if you need anything else.

**Comment by JPrause** [comment]
Reviewed after latest update. 👍

**Comment by cybette** [comment]
Thanks all! Will merge and share post tomorrow morning.

**Comment by Fryguy** [comment]
My only concern is that the dropbox account is not the "ManageIQ" account, and thus if the images are deleted, this post will not survive.  Perhaps it's better to just commit the images to this repo directly like we've done for other blog posts?

**Comment by syncrou** [comment]
@Fryguy,  @cybette - Images are all now local.

**Pull request merged**